### PR TITLE
Update configPage with CSS

### PR DESCRIPTION
**Pull request by StillLoading**
Implemented CSS where possible. Removed unused options and default values (this is done in PluginConfiguration.cs).

### CODE REVIEW

**Comment by Matt07211** [comment]
Just tested it, looks good, and seems to close the related issue here https://github.com/jellyfin/jellyfin-web/issues/77

**Comment by Matt07211** [comment]
Upon more testing I am unable to modify the value stored in AniDB_wait_time and there is no visual option on the web ui to toggle this option ```<AddAnimeGenre>true</AddAnimeGenre>``` that can be seen in the config.

File: Jellyfin.Plugin.Anime/Configuration/configPage.html [hunk 1]
```diff
@@ -6,47 +6,46 @@
...
+            <div data-role="content">
+                <div class="content-primary">
 
-                <form id="animeConfigurationForm">
-                    <ul class="ulForm" data-role="listview">
-                        <li>
-                            <label for="titleLanguage">Title Language:</label>
-                            <select id="titleLanguage" name="titleLanguage">
+                    <form id="animeConfigurationForm">
+                        <div class="selectContainer"
```
Comment (Matt07211) [inline Jellyfin.Plugin.Anime/Configuration/configPage.html]: It should be ```<div class="selectContainer">``` and not ```<div class="selectContainer"```

**Comment by StillLoading** [comment]
> Upon more testing I am unable to modify the value stored in AniDB_wait_time and there is no visual option on the web ui to toggle this option `<AddAnimeGenre>true</AddAnimeGenre>` that can be seen in the config.

I saw that option too but it was never included in the config page as far as I could tell. I even went back and looked at the Emby plugin and I could not find any indication of this option being user configurable. 
That's why I also omitted it here.

**Edit:** AniDB_wait_time didn't save because the value assignment was wrong `$('chkAniDB_wait_time').val()` should be `$('#chkAniDB_wait_time').val()`.

File: Jellyfin.Plugin.Anime/Configuration/configPage.html (hunk 1 above)
Comment (StillLoading) [reply Jellyfin.Plugin.Anime/Configuration/configPage.html]: Fixed with de24b4d

**Comment by Matt07211** [comment]
> it was never included in the config page as far as I could tell.

Shall we include it then?


I'll create an issue for it.

**Review by joshuaboniface — APPROVED** [review]

**Pull request merged**